ocp.py

```diff
@@ -119,8 +110,8 @@ def __init__(self, train, track, optsDict={}):
         powerMax = train.powerMax/totalMass if train.powerMax is not None else None
         powerMin = train.powerMin/totalMass if train.powerMin is not None else None
 
-        accMax = min(accInf, train.accMax if train.accMax is not None else np.inf)
-        accMin = max(-accInf, -abs(train.accMin) if train.accMin is not None else -np.inf)
+        accMax = min(accInf, train.accMax if train.accMax is not None else np.inf) if hasattr(train, 'accMax') else None
```

Comment:
When setting the acceleration limit you already check if `accMax `is `None` in which case you set the constraint to `accInf`. Hence, here we can simply write `accMax = train.accMax if hasattr(train, 'accMax') else None`

Reply:
Wait... I got rid of the min part in this way......

Reply:
Here's what I'd do, I would simplify this line with:

`accMax = train.accMax if hasattr(train, 'accMax') else None`

And then replace line 201 with:

`min(accInf, abs(accMax) if accMax is not None else np.inf)`

Reply:
Thanks for spotting this "salad" I reverted some changes and went back to None meaning no bound (similar to power and force)